app/assets/javascripts/sufia/batch_edit.js

```diff
@@ -79,14 +79,15 @@ function batch_edit_init () {
             },
             combine_requests: function (requests) {
                 var data = deserialize(requests[0].data.replace(/\+/g, " "));
+                var adata;
```

Comment:
That variable is being used in line 86 but it wasn't being declared. This PR declares it so that is local to the function rather than going to the global scope.